inheritance_ruby.rb

```diff
@@ -0,0 +1,87 @@
+module Validate_if_input_is_number
+  def is_number? number
+    return number.to_i.to_s == number
+  end
+end
+
+class Person
+  include Validate_if_input_is_number
+  attr_accessor :name_of_person , :status_of_person , :identifier_number , :age_of_person
+  def initialize()
+    name_of_person = ""
+    status_of_person = ""
+    identifier_number = ""
+    age_of_person = ""
+  end
+
+  public
+  def person
+    puts "Inside class #{self.class}"
+  end
+
+  def details_of_Person
+    puts "Enter name of Person:"
+    name_of_person = gets.chomp
+    puts "Enter the status of Person:"
+    status_of_person = gets.chomp
+    puts "Enter ID No. of Person:"
+    while true do
+      identifier_number = gets.chomp
+      if is_number? identifier_number
+        puts "ID No. of Person is : #{identifier_number}"
+        break
+      else
+        puts "Re-enter ID Number" 
+      end
+    end
+    puts "Enter age of Employee:"
+    while true do
+      age_of_person = gets.chomp
+      if is_number? age_of_person
```

Comment:
This method is too big. Break into smaller methods. Try to avoid puts statement by having different method which only puts the result.